README.rst

```diff
@@ -238,6 +212,66 @@ The parser use a ``TAG_KEY_MAPPING``, which one can override by calling ``rispy.
     'url',
     'volume']
 
+List tags can be customized in the same way, by passing a list to the ``list_tags`` parameter.
+
+Changing rispy behavior
+*********************
+
+There are a few flags that can be passed to ``rispy.load()`` and ``rispy.dump()`` that change how ``rispy`` deals with tags. For example, setting ``skip_unknown_tags`` to ``True`` will cause ``rispy`` do not read or write tags not in the tag map. More can be found in the docstrings for each class. If more customization is necessary, a custom implementation can be created (see next section).
+
+
+Using custom implementations
+----------------------------
+Not all RIS files follow the same formatting guidelines. There is an interface for creating custom implementations for reading and writing such files. An implementation contains the methods and parameters used to work with RIS files, and should be passed to ``rispy.load()`` or ``rispy.dump()``.
+
+
+Customizing implementations
+***************************
+Creating a custom implentation involves creating a class that inherits a base class, and overriding the necessary variables and methods. One of the existing parsers can also be inherited. Inheriting an existing class is advantageous if only minor changes need to be made. The sections below document what is available to be overriden, along with a few examples.
+
+Parsing
+^^^^^^^
+Custom parsers can inherit ``RisParser`` (the default parser) or ``BaseParser``. Various parameters and methods can be overridden when creating a new parser. These are documented in the ``BaseParser`` docstring.
+
+Examples:
+
+.. code:: python
+   
+   class WokParser(BaseParser):
+       """Subclass of Base for reading Wok RIS files."""
+
+       START_TAG = "PT"
+       IGNORE = ["FN", "VR", "EF"]
+       PATTERN = r"^[A-Z][A-Z0-9] |^ER\s?|^EF\s?"
+       DEFAULT_MAPPING = WOK_TAG_KEY_MAPPING
+       DEFAULT_LIST_TAGS = WOK_LIST_TYPE_TAGS
+
+       def get_content(self, line):
+           return line[2:].strip()
+
+       def is_header(self, line):
+           return True
+
+Writing
+^^^^^^^
+
+Writing is very similar to parsing. A custom writer class can inherit ``BaseWriter`` or ``RisWriter``.
```

Comment:
suggested edit: A custom writer class can inherit from  ``BaseWriter`` or one of it's subclasses such as ``RisWriter``.